dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBContextCache.java

```diff
@@ -0,0 +1,504 @@
+                        }
+                    });
+	            if (isLoggableFine) {
+	               long endTime = System.currentTimeMillis();
+	               int elapsedTime = (int)(endTime - startTime);
+	               LOG.fine("JAXBContext2 creation time in milliseconds: " + elapsedTime);
+	            }  
+		    // Liberty Change end
+                } catch (PrivilegedActionException e) {
+                    throw ex;
+                }
+            }
+            if (context == null) {
+                throw ex;
+            }
+        }
+        cachedContextAndSchemasInternal = new CachedContextAndSchemasInternal(context, classes);
+        synchronized (JAXBCONTEXT_CACHE) {
+            if (typeRefs == null || typeRefs.isEmpty()) {
+                if (cachedContextAndSchemasInternalMap == null) {
+                    cachedContextAndSchemasInternalMap
+                        = new CacheMap<>();
+                }
+                cachedContextAndSchemasInternalMap.put((defaultNs != null) ? defaultNs : "",
+                    cachedContextAndSchemasInternal);
+                JAXBCONTEXT_CACHE.put(classes, cachedContextAndSchemasInternalMap);
+            }
+        }
+
+        return new CachedContextAndSchemas(context, classes, cachedContextAndSchemasInternal);
+    }
+
+    private static boolean checkObjectFactoryNamespaces(Class<?> clz) {
+        for (Method meth : clz.getMethods()) {
+            XmlElementDecl decl = meth.getAnnotation(XmlElementDecl.class);
+            if (decl != null
+                && XmlElementDecl.GLOBAL.class.equals(decl.scope())
+                && StringUtils.isEmpty(decl.namespace())) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+
+    private static JAXBContext createContext(final Set<Class<?>> classes,
+                                      final Map<String, Object> map,
+                                      Collection<Object> typeRefs)
+        throws JAXBException {
+        boolean isLoggableInfo = LOG.isLoggable(Level.INFO);   // Liberty Change
+        JAXBContext ctx;
+        if (typeRefs != null && !typeRefs.isEmpty()) {
+            Class<?> fact = null;
+            String pfx = "com.sun.xml.bind.";
+            try {
+                fact = ClassLoaderUtils.loadClass("com.sun.xml.bind.v2.ContextFactory",
+                                                  JAXBContextCache.class);
+            } catch (Throwable t) {
+                try {
+                    fact = ClassLoaderUtils.loadClass("com.sun.xml.internal.bind.v2.ContextFactory",
+                                                      JAXBContextCache.class);
+                    pfx = "com.sun.xml.internal.bind.";
+                } catch (Throwable t2) {
+                    //ignore
+	            // Liberty Change begin
+	            if (isLoggableInfo) {
+	               LOG.info("Ignoring throwable t2: " + t2);  
```

Comment:
I don't think this is a good message for an info, should this be trace?

Reply:
I think it depends on "why" it is being ignored.   Unless it is truly unimportant, then skip the trace.